config/environments/test.rb

```diff
@@ -55,5 +55,5 @@
   # Raise error when a before_action's only/except options reference missing actions.
   config.action_controller.raise_on_missing_callback_actions = true
 
-  config.active_job.queue_adapter = :async
+  config.active_job.queue_adapter = :inline
```

Comment:
Inline plays nicely with the tests.

Reply:
Do you know why/when `async` started not playing nicely? I can imagine it being a bit risky baking the assumption that jobs run synchronously into the tests, but I don't have a specific objection.

Reply:
Now there are numerous actions that are triggered in tests that now result in mailer jobs being submitted. With async, those were being performed at unexpected times. Inline forces them to happen within the test that they were being submitted.

Async was probably a bad choice to begin with.